types/wlr_buffer.c

```diff
@@ -226,7 +205,12 @@ struct wlr_client_buffer *wlr_client_buffer_import(
 		wl_resource_post_error(resource, 0, "unknown buffer type");
```

Comment:
The `wlr_log` message above is outdated: this function doesn't upload textures anymore. We can probably just drop the "Cannot upload texture" part.